packages/design-system/src/styles/components/_Dialog.scss

```diff
@@ -54,6 +54,9 @@ dialog.fixed {
   box-sizing: border-box;
   color: var(--color-base); // Needed to override user-agent styles `canvasText`
   display: block;
+
+  // Reset so font size of ancestor elements don't affect the dialog
```

Comment:
Looks like our linter is automatically putting an extra line break above this comment. When I had the comment on the same line as the code, it made it even weirder.